Cargo.lock

```diff
@@ -3616,7 +3616,7 @@ dependencies = [
  "httpdate",
  "itoa",
  "pin-project-lite",
- "socket2 0.4.10",
```

Comment:
This has sadly been flapping between socket 0.4 and socket 0.5 for a long time :(